app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,3 @@
+class PostsController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] Great work creating the index action in this controller. However, there should also be a show action in this controller. This is a major requirement for this project to be approved.